forge-gui/res/cardsfolder/upcoming/glimmer_seeker.txt

```diff
@@ -0,0 +1,10 @@
+Name:Glimmer Seeker
+ManaCost:2 W
+Types:Creature Human Survivor
+PT:3/3
+T:Mode$ Phase | Phase$ Main2 | ValidPlayer$ You | IsPresent$ Card.Self+tapped | TriggerZones$ Battlefield | Execute$ TrigBranch | TriggerDescription$ Survival — At the beginning of your second main phase, if CARDNAME is tapped, draw a card if you control a Glimmer creature. If you don't control a Glimmer creature, create a 1/1 white Glimmer enchantment creature token.

```

Comment:
the condition is badly scripted because it wouldn't use LKI, see e.g. https://www.mtgsalvation.com/forums/magic-fundamentals/magic-rulings/817450-killing-kalamax-in-response-to-their-first-instant

if you want you can refactor others where it's relevant, or want me to do it?

Reply:
Hm, how exactly to add LKI here, please?